includes/abstracts/llms-abstract-admin-wizard.php

```diff
@@ -0,0 +1,390 @@
+			$extra = llms()->assets->enqueue_style( 'llms-admin-importer' );
+		}
+
+		return llms()->assets->enqueue_script( 'llms-admin-setup' ) && llms()->assets->enqueue_style( 'llms-admin-setup' ) && $extra;
+
+	}
+
+	/**
+	 * Retrieve the redirect URL to use after an import is complete at the conclusion of the wizard
+	 *
+	 * If a single course is imported, redirects to that course's edit page, otherwise redirects
+	 * to the course post table list sorted by created date with the most recent courses first.
+	 *
+	 * @since [version]
+	 *
+	 * @param int[] $course_ids WP_Post IDs of the course(s) generated during the import.
+	 * @return string
+	 */
+	protected function get_completed_url( array $course_ids ): string {
+
+		$count = count( $course_ids );
+
+		if ( 1 === $count ) {
+			return get_edit_post_link( $course_ids[0], 'not-display' );
+		}
+
+		return admin_url( 'edit.php?post_type=course&orderby=date&order=desc' );
+
+	}
+
+	/**
+	 * Retrieve the current step and default to the intro
+	 *
+	 * @since [version]
+	 *
+	 * @return string
+	 */
+	public function get_current_step(): string {
+		$step_keys = array_keys( $this->get_steps() );
+
+		return ( $_GET['step'] ?? '' ) ? llms_filter_input_sanitize_string( INPUT_GET, 'step' ) : $step_keys[0]; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	}
+
+	/**
+	 * Get slug if next step
+	 *
+	 * @since [version]
+	 *
+	 * @param string $step Step to use as current.
+	 * @return string
+	 */
+	public function get_next_step( string $step = '' ): string {
+		$step = $step ?: $this->get_current_step();
+		$keys = array_keys( $this->get_steps() );
+		$i    = array_search( $step, $keys, true );
+
+		// Next step doesn't exist or the next step would be greater than the index of the last step.
+		if ( false === $i || $i + 1 >= count( $keys ) ) {
+			return false;
+		}
+
+		return $keys[ ++$i ] ?? '';
+	}
+
+	/**
+	 * Get slug if prev step
+	 *
+	 * @since [version]
+	 *
+	 * @param string $step Step to use as current.
+	 * @return string
+	 */
+	public function get_prev_step( string $step = '' ): string {
+		$step = $step ?: $this->get_current_step();
+		$keys = array_keys( $this->get_steps() );
+		$i    = array_search( $step, $keys, true );
+
+		if ( false === $i || $i - 1 < 0 ) {
+			return false;
+		}
+
+		return $keys[ $i - 1 ] ?? '';
+	}
+
+	/**
+	 * Get the text to display on the "save" buttons
+	 *
+	 * @since [version]
+	 *
+	 * @param string $step Step to get text for.
+	 * @return string The translated text.
+	 */
+	private function get_save_text( string $step ): string {
+
+		/**
+		 * Filter the Save button text for a given step in the setup wizard
+		 *
+		 * The dynamic portion of this hook, `$step`, refers to the slug of the current step.
+		 *
+		 * @since [version]
+		 *
+		 * @param string $text Button text string.
+		 */
+		return apply_filters( "llms_setup_wizard_get_{$step}_save_text", $this->get_steps()[ $step ]['save'] ?? '' );
```

Comment:
```suggestion
		return apply_filters( "llms_{$this->type}_wizard_get_{$step}_save_text", $this->get_steps()[ $step ]['save'] ?? '' );
```

Also update the description to include a description for this other _dynamic portion_